frontend/app/src/composables/api/history/events/index.ts

```diff
@@ -37,10 +38,13 @@ import { getFilename } from '@/utils/file';
 
 interface QueryExchangePayload { name: string; location: string }
 
-interface EvmTransactionStatus {
-  lastQueriedTs: number;
-  pendingDecode: boolean;
-}
+const EvmTransactionStatusSchema = z.object({
+  hasEvmAccounts: z.boolean(),
+  lastQueriedTs: z.number(),
+  pendingDecode: z.boolean().optional(),
```

Comment:
the api was changed from `pendingDecode` to `undecodedTxCount` 
https://github.com/rotki/rotki/pull/10358